graphql-proxy/package.json

```diff
@@ -0,0 +1,58 @@
+{
+  "name": "events-helsinki-graphql-proxy",
+  "description": "Events Helsinki GraphQL proxy.",
+  "version": "0.0.1",
+  "private": true,
+  "main": "./src/index.ts",
+  "scripts": {
+    "build": "tsc -p . --outDir ./build",
+    "start": "nodemon",
+    "start:prod": "node build",
+    "test": "NODE_ENV=test PORT=0 jest",
+    "codegen": "gql-gen",
+    "lint": "eslint --ext js,ts,tsx src",
+    "format-code": "eslint --ext js,ts,tsx src --fix"
+  },
+  "dependencies": {
+    "@types/node": "^12.11.5",
+    "apollo-datasource-rest": "^0.6.6",
+    "apollo-server": "^2.9.7",
+    "apollo-server-express": "^2.9.7",
+    "cors": "^2.8.5",
+    "dotenv": "^8.2.0",
+    "express": "^4.17.1",
+    "graphql": "^14.5.8",
+    "graphql-depth-limit": "^1.1.0",
+    "lodash": "^4.17.15",
+    "typescript": "^3.6.4"
+  },
+  "devDependencies": {
+    "@graphql-codegen/cli": "^1.8.1",
+    "@graphql-codegen/typescript": "^1.8.1",
+    "@graphql-codegen/typescript-operations": "^1.8.1",
+    "@types/jest": "^24.0.19",
+    "@types/lodash": "^4.14.144",
+    "jest": "24.9.0",
+    "nodemon": "^1.19.4",
+    "ts-node": "^8.4.1",
+    "@typescript-eslint/eslint-plugin": "^2.5.0",
+    "@typescript-eslint/parser": "^2.5.0",
+    "babel-eslint": "^10.0.3",
+    "eslint": "^6.5.1",
+    "eslint-config-prettier": "^6.4.0",
+    "eslint-config-react-app": "^5.0.2",
+    "eslint-plugin-flowtype": "^3.13",
+    "eslint-plugin-import": "^2.18.2",
+    "eslint-plugin-jsx-a11y": "^6.2.3",
+    "eslint-plugin-prettier": "^3.1.1",
+    "eslint-plugin-react": "^7.16.0",
+    "eslint-plugin-react-hooks": "^1.7",
+    "eslint-plugin-simple-import-sort": "^4.0.0",
+    "prettier": "^1.18.2"
+  },
+  "jest": {
+    "setupFilesAfterEnv": [
+      "./test/jest.setup.js"
```

Comment:
This file does not exist, causing `yarn test` to fail.

Reply:
FIxed, setup file was on wrong folder